### PR TITLE
feature: init AieraChat

### PR DESCRIPTION
**Pull request by getnashty**
# Trello card link

# Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

## Acceptance

https://www.notion.so/aiera/Test-Flows-Template-5203736ccfcb424c81643065f2bbf5e2

- [ ] I have reviewed the Test Flows template and it is up-to-date after considering my changes

### CODE REVIEW

File: src/modules/AieraChat/components/Messages/MessageFactory/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { MessagePrompt } from './MessagePrompt';
+import { MessageResponse } from './MessageResponse';
+import { SourcesResponse } from './SourcesResponse';
+
+export const MessageFactory: VirtuosoMessageListProps<ChatMessage, MessageListContext>['ItemContent'] = ({
+    data,
+    context,
+}) => {
+    return match(data)
+        .with({ type: ChatMessageType.prompt }, (promptData) => <MessagePrompt data={promptData} />)
```
Comment (getnashty) [inline src/modules/AieraChat/components/Messages/MessageFactory/index.tsx:15]: update to all caps

File: src/modules/AieraChat/components/Messages/MessageFactory/MessagePrompt/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            >
+                <MicroQuestionMark className="w-4 text-white" />
+            </div>
+            <div className="self-center">
+                <p
+                    className={classNames('text-base font-bold antialiased leading-[1.125rem]', {
+                        'line-clamp-2': isStickyHeader,
+                    })}
+                >
+                    {searchTerm
```
Comment (getnashty) [inline src/modules/AieraChat/components/Messages/MessageFactory/MessagePrompt/index.tsx:47]: searchable text component

File: src/modules/AieraChat/components/Messages/MessageFactory/Block/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    source: string;
+    url?: string;
+    date?: string;
+    author?: string;
+};
+
+// Type for content that can include citations
+export type CitableContent = (string | Citation)[];
+
+export enum BlockType {
```
Comment (getnashty) [inline src/modules/AieraChat/components/Messages/MessageFactory/Block/index.tsx:23]: capitalize

File: src/modules/AieraChat/components/Messages/MessageFactory/Block/Chart/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+// Updated chart metadata types
+export type ChartMetaBase = {
+    title?: string;
+    xAxis?: string;
+    yAxis?: string;
+};
+
+export enum ChartType {
```
Comment (getnashty) [inline src/modules/AieraChat/components/Messages/MessageFactory/Block/Chart/index.tsx:27]: capitalize

File: src/modules/AieraChat/services/messages.ts [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+}
+
+interface ChatMessageSource {
+    targetType: string;
+    targetId: string;
+    targetTitle: string;
+}
+
+export enum ChatMessageStatus {
+    // Initial states
```
Comment (getnashty) [inline src/modules/AieraChat/services/messages.ts:17]: maybe this is on the session

**Review by mmichael0413 — APPROVED** [review]

**Pull request merged**